drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -516,6 +516,304 @@ def compare_ra_dec_crossmatches(hap_obj, json_timestamp=None, json_time_since_ep
+                        sep_stat_dict["Non-clipped min"] = np.min(sep)
+                        sep_stat_dict["Non-clipped max"] = np.max(sep)
+                        sep_stat_dict["Non-clipped mean"] = np.mean(sep)
+                        sep_stat_dict["Non-clipped median"] = np.median(sep)
+                        sep_stat_dict["Non-clipped standard deviation"] = np.std(sep)
+                        sigma = 3
+                        maxiters = 3
+                        clipped_stats = sigma_clipped_stats(sep, sigma=sigma, maxiters=maxiters)
+                        sep_stat_dict["{}x{} sigma-clipped mean".format(maxiters, sigma)] = clipped_stats[0]
+                        sep_stat_dict["{}x{} sigma-clipped median".format(maxiters, sigma)] = clipped_stats[1]
+                        sep_stat_dict["{}x{} sigma-clipped standard deviation".format(maxiters, sigma)] = clipped_stats[2]
+
+                        # Store statistics as new data section
+                        diag_obj.add_data_item(sep_stat_dict, "Interfilter cross-matched {} separation statistics".format(colname),
+                                               descriptions={"Non-clipped min": "Non-clipped min difference",
+                                                             "Non-clipped max": "Non-clipped max difference",
+                                                             "Non-clipped mean": "Non-clipped mean difference",
+                                                             "Non-clipped median": "Non-clipped median difference",
+                                                             "Non-clipped standard deviation": "Non-clipped standard deviation of differences",
+                                                             "3x3 sigma-clipped mean": "3x3 sigma-clipped mean difference",
+                                                             "3x3 sigma-clipped median": "3x3 sigma-clipped median difference",
+                                                             "3x3 sigma-clipped standard deviation": "3x3 sigma-clipped standard deviation of differences"},
+                                               units={"Non-clipped min": "pixels",
+                                                      "Non-clipped max": "pixels",
+                                                      "Non-clipped mean": "pixels",
+                                                      "Non-clipped median": "pixels",
+                                                      "Non-clipped standard deviation": "pixels",
+                                                      "3x3 sigma-clipped mean": "pixels",
+                                                      "3x3 sigma-clipped median": "pixels",
+                                                      "3x3 sigma-clipped standard deviation": "pixels"})
+        # TODO: should stats be in instrument-specific pixels or instrument-generic arcseconds??
```

Comment:
This is a good question.  Getting the images to combine correctly with 'drizzlepac', though, relies on image alignment to be good to better than 0.1 output pixels regardless of the size of the pixel on the sky.  Given that we may be comparing filters from different detectors, it may make sense to include the plate scale for the 'reference frame' as part of the summary information on the cross-matched catalogs.

Reply:
issue resolved by commit [f0bb5ad](https://github.com/spacetelescope/drizzlepac/pull/719/commits/f0bb5ad7c371aecf7bd615393618889fe2c1aa90)